book/website/project-design/documentation.md

```diff
@@ -0,0 +1,140 @@
+
+The reality is that documentation is an integral part of all research processes, from start to finish.
+
+<https://carpentries-incubator.github.io/managing-computational-projects/03-setup/index.html#provide-a-process-for-documentation>
+
+(pd-documentation-motivation)=
+## Motivation and Background
+
+A systematic process for documentation is more than a formal book-keeping practice because it:
+
+- allows everyone in your research to understand the research direction and track progress;
+- adds validity to your research work when systematically built on published peer-reviewed work;
+- communicates different ways to contribute, enabling diverse participation in the co-development;
+- upholds practices to ensure equity, diversity and inclusion;
+- recognises contributions fairly;
+- gives and shares credits for all work;
+- tracks the history of what worked or what did not work;
+- creates transparency about early and intermediate research outcomes;
+- makes auditing easy for funders, advisors or data managers;
+- helps reframe research narratives by connecting different work;
+- explains all decisions and stakeholders impacted by that;
+- gives the starting point for writing manuscript and publication; and more!
+
+<https://carpentries-incubator.github.io/managing-computational-projects/03-setup/index.html#a-systematic-process-for-documentation-is-more-than-a-formal-book-keeping-practice-because-it>
+
+## Documentation is a Shared Responsibility
+
+```{note}
+Whatever your approach is, be firm about making documentation a shared responsibility so that this job does not solely fall on the shoulders of early career researchers, members from traditionally marginalised groups or support staff.
+```
+
+The biggest question here is probably not ‘why’ but ‘how’ to facilitate documentation so that it is not challenging or burdensome for the team members.
+Here are a few recommendations to make documentation easier:
+
+- Allocate some time at the beginning of the project to discuss with the main stakeholders of the project about what should be documented.
+- Keep the tasks simple by establishing a shared repository for documentation with standard templates to guide how one should go about documenting their work (*It is always easier to start with a template than an empty sheet!*).
+- Add documentation sprint to your project timelines and milestones to make sure that everyone is aware of their importance in the project.
+- Create visible ways to recognise and incentivise the process of documenting.
+
+<https://carpentries-incubator.github.io/managing-computational-projects/03-setup/index.html#facilitating-documentation-in-your-team>
+
+(pd-documentation-types)=
+## Types of Documentation in Research and Data Science
+
+### Project Documentation
+
+[provide Summary]
+
+See {ref}`pd-project-repo<Creating Project Repositories>` chapter for details.
```

Comment:
```suggestion
See {ref}`Creating Project Repositories<pd-project-repo>` chapter for details.
```

https://the-turing-way.netlify.app/community-handbook/style/style-crossref.html#providing-an-alternative-title-for-the-references